docs/source/instruction_set_extensions.rst

```diff
@@ -586,10 +586,10 @@ General ALU Operations
 +=================+=========================+========================================================================+
 | **cv.abs**      | **rD, rs1**             | rD = rs1 < 0 ? –rs1 : rs1                                              |
 +-----------------+-------------------------+------------------------------------------------------------------------+
-| **cv.slet**     | **rD, rs1, rs2**        | rD = rs1 <= rs2 ? 1 : 0                                                |
+| **cv.slet**     | **rD, rs1, rs2**        | rD[0] = rs1 <= rs2 ? 1 : 0                                             |
 |                 |                         | Note: Comparison is signed                                             |
 +-----------------+-------------------------+------------------------------------------------------------------------+
-| **cv.sletu**    | **rD, rs1, rs2**        | rD = rs1 <= rs2 ? 1 : 0                                                |
+| **cv.sletu**    | **rD, rs1, rs2**        | rD[0] = rs1 <= rs2 ? 1 : 0                                             |
```

Comment:
same as above

Reply:
Hi @davideschiavone 
Yes you are right. I restore previous description for both SLET instructions.